lava-v2-jobs-from-api.py

```diff
@@ -96,10 +124,9 @@ def main(args):
                         exit(1)
             if build['kernel_image']:
                 # handle devices without a DTB, hacky :/
+                build['dtb_dir_data'].append('qemu')
```

Comment:
Yes to consolidation, just having a qemu device for v2 is nice

Maybe we actually do want a qemu boot on every defconfig but we haven't previously so this is a change of behaviour. I've actually turned off v2 boots on production as there's no point, but this would produce a lot more boot jobs for all labs when it does get turned on so it's something to discuss.